pages/nfttest/mint/_iscnId.vue

```diff
@@ -272,6 +274,79 @@ export default class NFTTestMintPage extends Vue {
     }
   }
 
+  async getOgImage() {
+    try {
+      const { url } = this.iscnData.contentMetadata
+      if (!url) { return }
+      const { data: { image } } = await this.$axios.get(`/crawler/?url=${encodeURIComponent(url)}`)
+      if (!image) { return }
+      const { data, headers } = await this.$axios.get(image)
+      const blob = new Blob([data], { type: headers['content-type'] })
+      const formData = new FormData()
+      formData.append('file', blob)
+      this.ogImageFormData = formData
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.error(err)
+    }
+  }
+
+  async estimateArweaveFee() {
+    try {
+      const { address, arweaveId, LIKE, memo } = await this.$axios.$post(
+        API_POST_ARWEAVE_ESTIMATE,
+        this.ogImageFormData,
+        {
+          headers: {
+            'Content-Type': 'multipart/form-data',
+          },
+        },
+      )
+      this.ogImageArweaveId = arweaveId
+      return {
+        to: address,
+        amount: new BigNumber(LIKE),
+        memo,
+      }
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.error('CANNOT_ESTIMATE_ARWEAVE_FEE')
+      throw err
+    }
+  }
+
+  async sendArweaveFeeTx({ to, amount, memo }: { to: string, amount: BigNumber, memo: string }): Promise<void> {
+    if (!this.signer) throw new Error('SIGNER_NOT_INITED')
+    if (!to) throw new Error('TARGET_ADDRESS_NOT_SET')
+    try {
+      const { transactionHash } = await sendLIKE(this.address, to, amount.toFixed(), this.signer, memo)
+      this.ogImageTxHash = transactionHash
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.error('CANNOT_SEND_ARWEAVE_FEE_TX')
+      throw err
+    }
+  }
+
+  async submitToArweave(): Promise<void> {
+    try {
+      const { arweaveId } = await this.$axios.$post(
+        `${API_POST_ARWEAVE_UPLOAD}?txHash=${this.ogImageTxHash}`,
```

Comment:
I think `ogImageTxHash` can be passed as function param instead of this.data here, you don't use the hash any where else.